### DISCUSSION TITLE
QuestionToStatementTranslator

### DISCUSSION THREAD

**Discussion by sudvijayr-nlp** [stanfordnlp/CoreNLP#1235] [Q&A]
QuestionToStatementTranslator is not generating statements for few specific questions, are these known issues?
Here is an example,
/usr/bin/java edu.stanford.nlp.naturalli.QuestionToStatementTranslator
[main] INFO edu.stanford.nlp.pipeline.StanfordCoreNLP - Adding annotator tokenize
[main] INFO edu.stanford.nlp.pipeline.StanfordCoreNLP - Adding annotator ssplit
[main] INFO edu.stanford.nlp.pipeline.StanfordCoreNLP - Adding annotator pos
[main] INFO edu.stanford.nlp.tagger.maxent.MaxentTagger - Loading POS tagger from edu/stanford/nlp/models/pos-tagger/english-left3words/english-left3words-distsim.tagger ... done [1.0 sec].
[main] INFO edu.stanford.nlp.pipeline.StanfordCoreNLP - Adding annotator lemma
question> who killed Tom?
question>

Questions like above don't generate responses. i'm checking the code, are these known issues?

**Comment by AngledLuffa** [comment]
I didn't even know this existed, and now that I do, I have no desire to attempt to work on it.  It looks like this is based on a bunch of deterministic rules, and simply put, they don't have enough coverage.  Here is a brief sample of how well it works...

I'll ask briefly if anyone knows of a different solution

```
question> Where was Tom born?
  -> Tom was born at location
question> Where did Tom die?
  -> Tom die at location
question> Who killed Tom?
question> Where were you when Tom was killed?
  -> were you when Tom was killed at location
question> How much wood could a woodchuck chuck if a woodchuck could chuck wood?
question> Does this actually do anything?
question> What state is north of New Jersey?
  -> north of New Jersey is state
question> What does duck taste like?
  -> duck taste like thing
question> Will the Flyers win this shootout?
question> Is this game over?
question> Are you holding on?
question> Are you depressed that she's gone?
question> What beautiful mistakes did you make?
  -> you make beautiful mistakes
question> Who is naked in my bed?
  -> naked in my bed is person
question> Who lies awake?
question> Would you take them back?
question> Why is Mox Opal banned?
question> Where can I meet Andorian women?
question> What color are Andorians?
  -> Andorians are color
question> What is this crap?
  -> thing is this crap
```

**Reply by sudvijayr-nlp** [reply]
Exactly it is based on few deterministic rules, namely thing, location, person, time. I will get it in a while and have it fixed. Meanwhile any suggestions will help.

**Comment by AngledLuffa** [comment]
If you have any suggestions for improvement, happy to hear them!
